api/ShadowRoot.json

```diff
@@ -307,10 +307,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": false
+              "version_added": "93"
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "93"
```

Comment:
With Safari, experimental can be `false`

Reply:
Thanks. I fixed this, and a couple of other cases in the docs - including the top level API.